packages/generator/src/enum-type/enum.ts

```diff
@@ -0,0 +1,81 @@
+/* Copyright (c) 2020 SAP SE or an SAP affiliate company. All rights reserved. */
```

Comment:
General: Search and replace for all `complex`. Should be removed, renamed or adjusted.

Reply:
same